src/SerialPort/ConflictingSettingsDialog.tsx

```diff
@@ -140,6 +145,7 @@ const DisplayConflictingSettings = ({
             }
         });
     }
+    allKeys.splice(allKeys.indexOf('path'), 1);
```

Comment:
I'd prefer not to mutate variables as it might be hard to debug in future. 
you can create a new variable by destructing object for the code below or code above